src/components/ProgressBar/ProgressBar.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <div
+    role="progressbar"
+    title="Progress"
+    aria-live="polite"
+    :aria-valuenow="percentage ? percentage : ''"
+    :aria-valuemin="percentage ? 0 : ''"
+    :aria-valuemax="percentage ? 100 : ''"
+    :aria-busy="percentage > 0 && percentage < 100"
```

Comment:
very small nit: should it be percentage >= 0?

Reply:
default percentage is 0 which is falsy and so the bar does not show at 0, so I thought this made sense